### PR TITLE
Created photo page for stepper

### PR DESCRIPTION
**Pull request by MickhaelSh**
Created photo page for stepper
phprev
prevph

### CODE REVIEW

File: tests/unit/containers/mentor-home-page/add-photo/AddPhoto.spec.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  it('should render error after add wrong file size', async () => {
+    const fakeFile = new File(['photo'], 'test-file.png', { type: 'image/png' })
+    Object.defineProperty(fakeFile, 'size', { value: 15_000_000 })
+
+    const input = screen.getByLabelText('becomeTutor.photo.button')
+    fireEvent.change(input, { target: { files: [fakeFile] } })
+    const error = screen.queryByText('becomeTutor.photo.fileSizeError')
+
+    await waitFor(() => expect(error).toBeInTheDocument())
+    photo.push(fakeFile)
```
Comment (rostyslaw27) [inline tests/unit/containers/mentor-home-page/add-photo/AddPhoto.spec.js:69]: why did you push fakeFile to photo after waitFor?

Comment (MickhaelSh) [reply tests/unit/containers/mentor-home-page/add-photo/AddPhoto.spec.js]: changed to separate describe

**Review by rostyslaw27 — APPROVED** [review]

File: src/containers/mentor-home-page/add-photo/add-photo.style.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    width: '649px',
+    height: '700px',
+    alignItems: 'center',
+    border: 'dashed',
+    borderColor: 'primary.200',
+    borderRadius: '20px',
+    mr: 3
+  },
+  img: {
+    width: '649px',
```
Comment (maarekua) [inline src/containers/mentor-home-page/add-photo/add-photo.style.js:18]: Fixed height width is a bad idea.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/mentor-home-page/add-photo/add-photo.style.js (hunk 2 above)
Comment (MickhaelSh) [reply src/containers/mentor-home-page/add-photo/add-photo.style.js]: changed

File: src/containers/mentor-home-page/add-photo/AddPhoto.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box, Typography } from '@mui/material'
+
+import FileUploader from '~/components/file-uploader/FileUploader'
+
+import { style } from '~/containers/mentor-home-page/add-photo/AddPhoto.style'
+
+const validationData = {
+  maxFileSize: 10_000_000,
```
Comment (maarekua) [inline src/containers/mentor-home-page/add-photo/AddPhoto.js:10]: 10MB....

Comment (MickhaelSh) [reply src/containers/mentor-home-page/add-photo/AddPhoto.js:10]: done

**Review by dmytruk-y — APPROVED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Pull request merged**